client/app/services/vm-details/vm-details.html

```diff
@@ -37,9 +37,15 @@
         </custom-button>
         <custom-dropdown class="custom-dropdown pull-left"
                          config="item"
+                         ng-repeat="item in vm.snapshotListActions"
+                         menu-right="true">
+        </custom-dropdown>
+        <custom-dropdown class="custom-dropdown pull-left"
+                         config="item"
                          ng-repeat="item in vm.listActions"
                          menu-right="true">
         </custom-dropdown>
+       
```

Comment:
Is complaining about this unnecessary too pedantic? Unsure, gonna approve anyways, BUT ya should rebase this unnecessary space to OBLIVION 😁